simulator-core/src/test/java/com/evbox/everon/ocpp/it/availability/ChangeAvailabilityEvseIt.java

```diff
@@ -53,34 +46,26 @@ void shouldChangeEvseStatusToUnavailable() {
 
             ocppMockServer.verify();
         });
-
     }
 
     @Test
     void shouldChangeEvseStatusToAvailable() {
 
-        // given
         ocppMockServer
-                .when(bootNotificationRequest())
-                .thenReturn(bootNotificationResponse());
+                .when(StatusNotification.request(AVAILABLE), times(2))
```

Comment:
I think there is a method called - twice()

Reply:
we do, but we already use times(2) in our unit tests (because mockito doesn't have twice() method, so I think it should be consistent here)